### PR TITLE
docs/Improve Docker Documentation

### PR DESCRIPTION
**Pull request by sabakhilji**
issue# 1910

### CODE REVIEW

File: docs/content/quickstart/quick-start-docker.md [hunk 1]
```diff
@@ -48,17 +51,17 @@ SENDGRID_API_KEY="some_sendgrid_key"
...
 
-5. Start your server
+### 5. Start your server
 
 ```bash
 docker-compose up --build
 ```
 
-We will use docker-compose and Docker to start up our development environment. Running the above command will do the following:
+Running the above command does the following:
```
Comment (shahednasser) [inline docs/content/quickstart/quick-start-docker.md:60]: Add the following note before this:

~~~
:::note

If you get the error `ERROR: for postgres  Cannot start service postgres: Ports are not available`, change the ports used for PostgreSQL in `docker-compose.yml` to something like this:

```yml
  postgres:
    ports:
      - "5433:5433"
```

:::
~~~

**Review by shahednasser — CHANGES_REQUESTED** [review]
Thank you @sabakhilji for your contribution! I added a few comments for changes below

File: docs/content/quickstart/quick-start-docker.md [hunk 2]
```diff
@@ -1,9 +1,12 @@
...
-# Quickstart w. Docker
+# Quickstart using Docker
+In this document you will learn how to make a container of Medusa's app on Docker. Docker is an open source platform for building, deploying, and managing containerized applications.
 
-This quick start is intended for developers, that have already configured their local development environment and familiarised them selves with all the technologies and frameworks used throughout the Medusa eco-system.
 
-If this is not the case, please head over to our Getting Started tutorial for a thorough walkthrough.
+## Prerequisites
+This quick start is intended for developers, assuming that you have already configured your local development environment and familiarised yourselves with all the technologies and frameworks used throughout the Medusa eco-system.
+If this is not the case, please head over to our [Getting Started](https://docs.medusajs.com/quickstart/quick-start) tutorial for a thorough walkthrough.
```
[the remark this answers is not in the document]
Comment (sabakhilji) [reply docs/content/quickstart/quick-start-docker.md:8]: @shahednasser Do you want to delete both paragraphs "this quick start is intended.................................................  eco-system "and "If this is not the case................................walkthrough" please confirm.

**Comment by sabakhilji** [comment]
@shahednasser please review the required changes and tell me if anything else is required. Thanks!

File: docs/content/quickstart/quick-start-docker.md [hunk 3]
```diff
@@ -1,9 +1,19 @@
...
+# Quickstart using Docker
 
-If this is not the case, please head over to our Getting Started tutorial for a thorough walkthrough.
+In this document you will learn how to make a container of Medusa's app on Docker. Docker is an open source platform for building, deploying, and managing containerized applications.
+
+## Prerequisites
+
+### Node.js
+Medusa supports Node versions 14 and 16. You can check which version of Node you have by running the following command:
+```bash
```
Comment (shahednasser) [inline docs/content/quickstart/quick-start-docker.md:10]: Can you add a new line before this? Same with a new line after the closing backticks

File: docs/content/quickstart/quick-start-docker.md [hunk 4]
```diff
@@ -1,9 +1,19 @@
...
+node -v
+```
+You can install Node from the [official website](https://nodejs.org/en/).
+
+### Docker Desktop
```
Comment (shahednasser) [inline docs/content/quickstart/quick-start-docker.md:15]: Add a new line after this

File: docs/content/quickstart/quick-start-docker.md [hunk 5]
```diff
@@ -1,9 +1,12 @@
...
 
-This quick start is intended for developers, that have already configured their local development environment and familiarised them selves with all the technologies and frameworks used throughout the Medusa eco-system.
 
-If this is not the case, please head over to our Getting Started tutorial for a thorough walkthrough.
+## Prerequisites
+This quick start is intended for developers, assuming that you have already configured your local development environment and familiarised yourselves with all the technologies and frameworks used throughout the Medusa eco-system.
+If this is not the case, please head over to our [Getting Started](https://docs.medusajs.com/quickstart/quick-start) tutorial for a thorough walkthrough.
+It is assumed that you have docker installed on you system. If that is not the case please go and install [Docker](https://docs.docker.com/get-docker/).
 
 ## Introduction
```
[the remark this answers is not in the document]
Comment (shahednasser) [reply docs/content/quickstart/quick-start-docker.md:18]: I meant here to remove the Introduction section @sabakhilji

File: docs/content/quickstart/quick-start-docker.md [hunk 6]
```diff
@@ -15,75 +25,87 @@ With all the tools and technologies in place, let's get started by setting up a
...
-   - A variant of that product that costs 195 EUR
 
-Once done, our server will be accessible at `http://localhost:9000`.
+## Test Your Server
 
-## Try it out
+Once done, your server will be accessible at `http://localhost:9000`.
 
-Let's try out our Medusa server by fetching some products.
+You can test out your server using tools like Postman or sending a cURL request:
```
Comment (shahednasser) [inline docs/content/quickstart/quick-start-docker.md:77]:
```suggestion
You can test out your server using tools like Postman or by sending a cURL request:
```

Sorry I made a small mistake on this one

File: docs/content/quickstart/quick-start-docker.md [hunk 7]
```diff
@@ -15,75 +25,87 @@ With all the tools and technologies in place, let's get started by setting up a
...
 curl -X GET localhost:9000/store/products | python -m json.tool
 ```
 
-## What's next?
+:::note
+This command uses Python to format the result of the request better in your command line. If you don't want to use Python you can simply send a request without the formatting:
+```bash
+curl localhost:9000/store/products
+```
+:::
```
Comment (shahednasser) [inline docs/content/quickstart/quick-start-docker.md:88]: Add a new line before this

File: docs/content/quickstart/quick-start-docker.md [hunk 8]
```diff
@@ -15,75 +25,87 @@ With all the tools and technologies in place, let's get started by setting up a
...
+You can test out your server using tools like Postman or sending a cURL request:
 
 ```bash
 curl -X GET localhost:9000/store/products | python -m json.tool
 ```
 
-## What's next?
+:::note
+This command uses Python to format the result of the request better in your command line. If you don't want to use Python you can simply send a request without the formatting:
+```bash
```
Comment (shahednasser) [inline docs/content/quickstart/quick-start-docker.md:85]: add a new line before this

File: docs/content/quickstart/quick-start-docker.md [hunk 9]
```diff
@@ -1,9 +1,19 @@
-<!-- vale off -->
-# Quickstart w. Docker
```
Comment (shahednasser) [inline docs/content/quickstart/quick-start-docker.md:1]: Remove this empty line

**Review by shahednasser — CHANGES_REQUESTED** [review]
Thanks @sabakhilji just a few more changes.

Additionally, can you add a sidebar item for this document? It should be added in `www/docs/sidebar.js` just after the current Quickstart link.

**Comment by sabakhilji** [comment]
@shahednasser please review the changes . Thanks!

File: docs/content/quickstart/quick-start-docker.md (hunk 9 above)
Comment (shahednasser) [reply docs/content/quickstart/quick-start-docker.md:1]: @sabakhilji there's an empty line at the very beginning of the documentation please remove it

File: www/docs/sidebars.js [hunk 10]
```diff
@@ -30,6 +30,11 @@ module.exports = {
           id: "quickstart/quick-start",
           label: "Quickstart Guide",
         },
+       {
+          type:"doc",
+          id: "quickstart w Docker",
+          label :"Quickstart Docker",
+       },
```
Comment (shahednasser) [inline www/docs/sidebars.js:37]: Please fix the indentation here to match the sibling items

**Comment by sabakhilji** [comment]
done please check

File: www/docs/sidebars.js [hunk 11]
```diff
@@ -30,6 +30,11 @@ module.exports = {
           id: "quickstart/quick-start",
           label: "Quickstart Guide",
         },
+        {
+          type:"doc",
+          id: "quickstart w Docker",
```
Comment (shahednasser) [inline www/docs/sidebars.js:35]: for the id property the value should be `quickstart/quick-start-docker`

File: www/docs/sidebars.js [hunk 12]
```diff
@@ -30,6 +30,11 @@ module.exports = {
...
+          label :"Quickstart Docker",
```
Comment (shahednasser) [inline www/docs/sidebars.js:36]: Remove the label.

**Review by shahednasser — CHANGES_REQUESTED** [review]

**Review by shahednasser — APPROVED** [review]

**Comment by shahednasser** [comment]
Amazing work @sabakhilji thank you 🚀

**Comment by sabakhilji** [comment]
thanks, @shahednasser for the guidance and support

**Comment by sabakhilji** [comment]
done

**Pull request merged**